ChangeLog

```diff
@@ -4,6 +4,19 @@
     - Bug fixes
     * Others
 
+## 0.18.0 - 2018-07-17
+    + Support theme supplied avatars
```

Comment:
nitpick add dash line under version and date

Reply:
i assume the list of features for this release was checked with git log and is accurate